From 6cb9f0430a27a48694968177875d4568fd29084b Mon Sep 17 00:00:00 2001
From: Tomas Henzl <thenzl@redhat.com>
Date: Mon, 8 Sep 2014 14:31:34 -0400
Subject: [scsi] hpsa: remove unused kthread.h header

Message-id: <1410186734-19986-44-git-send-email-thenzl@redhat.com>
Patchwork-id: 92056
O-Subject: [RHEL6.6 e-stor PATCH 43/83] hpsa: remove unused kthread.h header
Bugzilla: 1069185
RH-Acked-by: Maurizio Lombardi <mlombard@redhat.com>
RH-Acked-by: David Milburn <dmilburn@redhat.com>
RH-Acked-by: Ewan Milne <emilne@redhat.com>

Signed-off-by: Stephen M. Cameron <scameron@beardog.cce.hp.com>
Signed-off-by: James Bottomley <JBottomley@Parallels.com>
(cherry picked from commit 9f02e5bc60d699017bc1f5e5cc95c4fedc7b29ba)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/hpsa.c b/drivers/scsi/hpsa.c
index f1e75d7..07068ce 100644
--- a/drivers/scsi/hpsa.c
+++ b/drivers/scsi/hpsa.c
@@ -47,7 +47,6 @@
 #include <linux/string.h>
 #include <linux/bitmap.h>
 #include <linux/atomic.h>
-#include <linux/kthread.h>
 #include <linux/jiffies.h>
 #include <asm/div64.h>
 #include "hpsa_cmd.h"
-- 
1.7.1